js/config/queries.js

```diff
@@ -27,7 +27,7 @@ export const GET_ITEMS = gql`
       location {
```

Comment:
I remember I had to add an id here for allItems. Please push corrections right away. Thank you.